src/collector/top_collector.rs

```diff
@@ -175,6 +180,26 @@ impl<T: PartialOrd + Clone> TopSegmentCollector<T> {
     }
 }
 
+fn stable_sort<T, D>(a: &ComparableDoc<T, D>, b: &ComparableDoc<T, D>) -> Ordering
```

Comment:
I usually name that kind of argument `left` and `right`. A lot of ppl use `lhs`, and `rhs` (as left hand side, and right hand side). Let's avoid `a` and `b` anyway.

Reply:
(this code doesn't exist anymore if we go for V2)